src/pages/CardCart.jsx

```diff
@@ -21,15 +25,15 @@ const CardCart = () => {
                             <div className="cart-sub-cont2">
                                 <div className="cart-info-qty">Quantity:</div>
                                 <div className="cart-qty-btn">
-                                    <button className="btn-cart-qty">-</button>
+                                    <button className="btn-cart-qty" onClick={() => dispatch_Cart({ type: "SUB_ITEM_QTY", payload: product })}>-</button>
                                     <input
                                         type="text"
                                         name=""
                                         id=""
```

Comment:
You can either give some values to these attributes or remove them.
Query -  You are not using the label for the input so can you explain what does this id attribute does here?